src/agent/onefuzz-agent/src/tasks/merge/generic.rs

```diff
@@ -19,13 +19,6 @@ use std::{
 use storage_queue::{QueueClient, EMPTY_QUEUE_DELAY};
 use tokio::process::Command;
 
-#[derive(Debug, Deserialize)]
-struct QueueMessage {
```

Comment:
This appears to have really never been used (since the initial release). Maybe the `Deserialize` proc macro was making it spuriously appear used.